examples/flask-kitchensink/app.py

```diff
@@ -332,6 +333,137 @@ def handle_text_message(event):
             event.reply_token,
             message
         )
+    elif text == 'flex_update_1':
+        bubble_string = """
```

Comment:
[ask]
Is there any reason to define as raw JSON?

[imo]
From the viewpoint of providing examples, it is better to define by using models SDK provides like the following:

```python
        bubble = BubbleContainer(
            body=BoxComponent(
                layout="vertical",
                padding_all="0px",
                contents=[
                    ImageComponent(
                        url="https://line-objects-dev.com/flex/bg/eiji-k-1360395-unsplash.jpg",
                        position="relative",
                        size="full",
                        aspect_mode="cover",
                        aspect_ratio="1:1",
                        gravity="center"
                    ),
                    BoxComponent(
                        layout="horizontal",
                        position="absolute",
                        padding_all="20px",
                        offset_bottom="0px",
                        offset_start="0px",
                        offset_end="0px",
                        background_color="#00000099",
                        contents=[
                            BoxComponent(
                                layout="vertical",
                                contents=[
                                    TextComponent(
                                        text="Brown Hotel",
                                        weight="bold",
                                        size="xl",
                                        color="#ffffff"
                                    ),
                                    BoxComponent(
                                        layout="baseline",
                                        margin="md",
                                        contents=[
                                            IconComponent(
                                                size="sm",
                                                url="https://scdn.line-apps.com/n/channel_devcenter"
                                                    "/img/fx/review_gold_star_28.png"
                                            ),
                                            IconComponent(
                                                size="sm",
                                                url="https://scdn.line-apps.com/n/channel_devcenter"
                                                    "/img/fx/review_gold_star_28.png"
                                            ),
                                            IconComponent(
                                                size="sm",
                                                url="https://scdn.line-apps.com/n/channel_devcenter"
                                                    "/img/fx/review_gold_star_28.png"
                                            ),
                                            IconComponent(
                                                size="sm",
                                                url="https://scdn.line-apps.com/n/channel_devcenter"
                                                    "/img/fx/review_gold_star_28.png"
                                            ),
                                            IconComponent(
                                                size="sm",
                                                url="https://scdn.line-apps.com/n/channel_devcenter"
                                                    "/img/fx/review_gray_star_28.png"
                                            ),
                                            TextComponent(
                                                size="sm",
                                                text="4.0",
                                                color="#d6d6d6",
                                                margin="md",
                                                flex=0,
                                            )
                                        ]
                                    )
                                ]
                            ),
                            BoxComponent(
                                layout="vertical",
                                contents=[
                                    TextComponent(
                                        text="¥62,000",
                                        color="#a9a9a9",
                                        decoration="line-through",
                                        align="end"
                                    ),
                                    TextComponent(
                                        text="¥42,000",
                                        color="#ebebeb",
                                        size="xl",
                                        align="end"
                                    )
                                ]
                            )
                        ]
                    ),
                    BoxComponent(
                        layout="vertical",
                        position="absolute",
                        background_color="#ff2600",
                        corner_radius="20px",
                        padding_all="5px",
                        offset_top="10px",
                        offset_end="10px",
                        padding_start="10px",
                        padding_end="10px",
                        contents=[
                            TextComponent(
                                text="SALE",
                                color="#ffffff"
                            )
                        ]
                    )
                ]
            )
        )
```

Reply:
[memo]
https://github.com/line/line-bot-sdk-python/pull/206#pullrequestreview-282329512